sdk/cosmos/azure-cosmos-spark_3-0_2-12/Samples/Scala Sample.scala

```diff
@@ -0,0 +1,69 @@
+// Databricks notebook source
+// configuration
+val cosmosEndpoint = "https://REPLACEME.documents.azure.com:443/"
+val cosmosMasterKey = "REPLACEME"
+val cosmosDatabaseName = "sampleDB"
+val cosmosContainerName = "sampleContainer"
+
+val cfg = Map("spark.cosmos.accountEndpoint" -> cosmosEndpoint,
+  "spark.cosmos.accountKey" -> cosmosMasterKey,
+  "spark.cosmos.database" -> cosmosDatabaseName,
+  "spark.cosmos.container" -> cosmosContainerName
+)
+
+val cfgWithAutoSchemaInferance = Map("spark.cosmos.accountEndpoint" -> cosmosEndpoint,
+  "spark.cosmos.accountKey" -> cosmosMasterKey,
+  "spark.cosmos.database" -> cosmosDatabaseName,
+  "spark.cosmos.container" -> cosmosContainerName,
+  "spark.cosmos.read.inferSchemaEnabled" -> "true"                          
+)
+
+// COMMAND ----------
+
+// create Cosmos Database and Cosmos Container using Catalog APIs
+spark.conf.set(s"spark.sql.catalog.cosmosCatalog", "com.azure.cosmos.spark.CosmosCatalog")
+spark.conf.set(s"spark.sql.catalog.cosmosCatalog.spark.cosmos.accountEndpoint", cosmosEndpoint)
+spark.conf.set(s"spark.sql.catalog.cosmosCatalog.spark.cosmos.accountKey", cosmosMasterKey)
+
+// create a cosmos database
+spark.sql(s"CREATE DATABASE IF NOT EXISTS cosmosCatalog.${cosmosDatabaseName};")
+
+// create a cosmos container
+spark.sql(s"CREATE TABLE IF NOT EXISTS cosmosCatalog.${cosmosDatabaseName}.${cosmosContainerName} using cosmos.items " +
+      s"TBLPROPERTIES(partitionKeyPath = '/id', manualThroughput = '1100')")
+
+// COMMAND ----------
+
+// ingestion
+spark.createDataFrame(Seq(("cat-alive", "Schrodinger cat", 2, true), ("cat-dead", "Schrodinger cat", 2, false)))
+  .toDF("id","Name","Age","isAlive")
+   .write
+   .format("cosmos.items")
+   .options(cfg)
+   .mode("APPEND")
+   .save()
+
+// COMMAND ----------
+
+// Show the schema of the table and data without auto schema inference
+val df = spark.read.format("cosmos.items").options(cfg).load()
+df.printSchema()
+
+df.show()
```

Comment:
nit: would it be worth showing how to see the raw JSON document like we have in the E2E tests?

Reply:
that's already included, there are two samples with and without auto schema inference.